### PR TITLE
Ports belt resprites and adds a Chief Medical Officer belt

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Ports some belt sprite PRs from Bee/TG and adds some coder sprites of my own:

- ["The Great Resprite: Atomized Edition (Tools volume)" #2487](https://github.com/BeeStation/BeeStation-Hornet/pull/2487) by [SomeAngryMiner](https://github.com/SomeAngryMiner) - Belt sprites
- [ "Medical belt resprite" #6202](https://github.com/BeeStation/BeeStation-Hornet/pull/6202) by [ImSynthex](https://github.com/ImSynthex)
- ["Belt sprites for the syndie screwdriver and alien tools" #65400](https://github.com/tgstation/tgstation/pull/65400) by [Ebb-Real](https://github.com/Ebb-Real) - Heavily altered belt overlay sprites
- ["Adds the CMO belt. Rework the CMO's roundstart gear." #59381](https://github.com/tgstation/tgstation/pull/59381) by [KathyRyals](https://github.com/KathyRyals) - Concept for the Chief Medical Officer's belt and what to put in it
- Coder sprited overlays for adv/alien surgical tools

New CMO belt starts with experimental tools, their advanced health analyzer, a crew pinpointer/monitor and surgical drapes. CMOs start with this when they spawn, and the duplicate items have been removed from their locker. If people have issues with them starting with these tools then I can split it off into its own PR and make it start empty like i originally had it.

image
image
image
image

# Wiki Documentation

Belt images will need to be updated.

# Changelog

:cl:  SomeAngryMiner, ImSynthex, Ebb-Real, KathyRyals
rscadd: Added Chief Medical Officer belt, which the CMO starts equipped with and uses soulless codersprites
imageadd: added new soulless sprites for the utility belt, CE's belt, and medical belt
imageadd: added belt overlays for abductor tools and surgical tools  
bugfix: CMO now starts with a skirt if their player chooses to start with one
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -205,6 +206,37 @@
...
+	new /obj/item/scalpel/alien(src)
+	new /obj/item/hemostat/alien(src)
+	new /obj/item/retractor/alien(src)
+	new /obj/item/circular_saw/alien(src)
+	new /obj/item/surgicaldrill/alien(src)
+	new /obj/item/cautery/alien(src)
+	new /obj/item/bonesetter(src)
+	new /obj/item/surgical_drapes(src)
+	new /obj/item/healthanalyzer/advanced(src)
+	new /obj/item/reagent_containers/pill/adminordrazine(src)
```
Comment (ToasterBiome) [inline code/game/objects/items/storage/belt.dm]: what is this

Comment (nmajask) [reply code/game/objects/items/storage/belt.dm]: /obj/item/storage/belt/medical/chief/**admin**/full/PopulateContents()

Just a separate toolbelt that admins can spawn in with abductor tools and adminordrazine for messing around, though neither of the full CMO toolbelts are used

Comment (nmajask) [reply code/game/objects/items/storage/belt.dm]: Removed it

**Comment by JamieD1** [comment]
@grimm-hollowknight

**Review by nesisknesis — APPROVED** [review]

**Comment by Sniblet** [comment]
Alien crowbars are now invisible

**Pull request merged**